src/main/java/com/github/alina/sesion4_challenge/ChallengeEighteen.java

```diff
@@ -0,0 +1,24 @@
+package com.github.alina.sesion4_challenge;
+//Write a Java program where you declare an integer array with 5 elements. Use a for loop to initialize the array such that each element is equal to its index incremented by 1. Then use another for loop to decrement each element by 1. Print the array before and after the decrement operation using a for-each loop
+public class ChallengeEighteen {
+    public static void main(String[] args) {
+        int[] array = new int[5];
+
+        // Initialize the array elements using a for loop
```

Comment:
Please do not comment on your code in the future, on each logic step. The code should explain itself, without comments. If there is really a need for a comment, we can either use java doc to document a method for its purpose.

Reply:
Remove all the comments from the challenges. Thanks